packages/hub/src/session.ts

```diff
@@ -63,7 +63,7 @@ export class Session implements SessionInterface {
     if (typeof context.started === 'number') {
       this.started = context.started;
     }
-    if (typeof context.duration === 'number') {
+    if (typeof context.duration === 'number' || typeof context.duration === undefined) {
```

Comment:
Hmm we need to think this through.
Seems problematic.

If you call `.update({timestamp: Date.now()/1000})` we should not set the duration to `undefined`.